src/backend/distributed/commands/vacuum.c

```diff
@@ -73,13 +76,24 @@ static CitusVacuumParams VacuumStmtParams(VacuumStmt *vacstmt);
 void
 PostprocessVacuumStmt(VacuumStmt *vacuumStmt, const char *vacuumCommand)
 {
-	int relationIndex = 0;
-	List *vacuumRelationList = ExtractVacuumTargetRels(vacuumStmt);
-	List *relationIdList = NIL;
 	CitusVacuumParams vacuumParams = VacuumStmtParams(vacuumStmt);
 	LOCKMODE lockMode = (vacuumParams.options & VACOPT_FULL) ? AccessExclusiveLock :
 						ShareUpdateExclusiveLock;
-	int executedVacuumCount = 0;
+
+
+	if (vacuumParams.options & VACOPT_VACUUM)
+	{
```

Comment:
maybe add a comment on why we have this if statement

Reply:
I actually added that inside the if block, do you think we should move it outside the if block?

Reply:
the comment makes it clear why we're doing CommitTransactionCommand, but not why we're not doing it if the VACOPT_VACUUM flag is set.